spec/functional/resource/windows_pagefile_spec.rb

```diff
@@ -40,13 +40,23 @@
     new_resource
   end
 
+  def set_automatic_managed_to_false
+    powershell_code = <<~EOH
+      $computersys = Get-WmiObject Win32_ComputerSystem -EnableAllPrivileges;
+      $computersys.AutomaticManagedPagefile = $False;
+      $computersys.Put();
+    EOH
+    powershell_exec!(powershell_code)
+  end
+
   describe "Setting Up Pagefile Management" do
     context "Disable Automatic Management" do
-      it "Disables Automatic Management" do
+      it "Verifies Automatic Management is Disabled" do
+        set_automatic_managed_to_false
```

Comment:
seems like it might be better to have it set auto management to true and then se if this is updated. That way you test that it can actually turn it off. Even better, keep this test as is and add one that sets it to true and then applies the resource to disable it.

Reply:
Done, works great and it actually uncovered another bug in the following test which I also corrected here. Woot.